README.md

```diff
@@ -10,22 +10,68 @@
 
 ```elm
 {-| Request to the server
+The genre and featureSet attributes are identifiers refering to a predefined
+setting.
 
 Example JSON:
-{ "knownAuthorText": "lorem", "unknownAuthorText": "ipsum" }
+{ "knownAuthorTexts": [ "lorem", "Hello World!" ]
+, "unknownAuthorText": "ipsum"
+, "language": "EN"
+, "genre": 0
+, "featureSet": 0
+}
 
 -}
-type alias ToServer =
-    { knownAuthorText : String, unknownAuthorText : String }
+type alias ToServerAttribution =
+    { knownAuthorTexts : String[]
+    , unknownAuthorText : String
+    , language : String
+    , genre : number
+    , featureSet : number
+    }
 
 
 {-| Response from the server
 
 Example JSON:
-{ "sameAuthor": true, "confidence": 0.67 }
+{ "sameAuthorConfidence": 0.67 }
 
 -}
-type alias FromServer =
-    { sameAuthor : Bool, confidence : Float }
+type alias FromServerAttribution =
+    { sameAuthorConfidence : Float }
+```
+
+### Author Profiling
+
+**endpoint**: /api/profiling
+
+```elm
+{-| Request to the server
+The genre and featureSet attributes are identifiers refering to a predefined
+setting.
+
+Example JSON:
+{ "text": "lorem ipsum"
+, "language": "EN"
+, "genre": 0
+, "featureSet": 0 }
+
+-}
+type alias ToServerProfiling =
+    { text : String
+    , language : String
+    , genre : number
+    , featureSet : number }
+
+
+{-| Response from server
+
+Example JSON:
+{ "age": "20-30"
+, "gender": "M" }
+
+-}
+type alias FromServerProfiling =
+    { age: String, gender: String }
```

Comment:
can we represent age as a tuple (or, in TS, fixed-size array)? I like that better